# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/websockets/stream/tentative/backpressure-send.any.js.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/websockets/stream/tentative/backpressure-send.any.js.ini
--- a/testing/web-platform/meta/websockets/stream/tentative/backpressure-send.any.js.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/websockets/stream/tentative/backpressure-send.any.js.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -20,11 +20,8 @@
 
 [backpressure-send.any.sharedworker.html?wpt_flags=h2]
   expected:
-    if (os == "win") and (processor == "x86") and debug: OK
-    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if (os == "android") and debug and not webrender: ["TIMEOUT", "ERROR"]
-    if (os == "win") and (processor == "x86_64"): OK
-    TIMEOUT
+    if (os == "android") and debug and not webrender: ["OK", "ERROR", "TIMEOUT"]
+    if (os == "win") and (processor == "x86") and not debug: ["OK", "TIMEOUT"]
   [backpressure should be applied to sent messages]
     expected: FAIL
 
@@ -51,35 +48,26 @@
 
 [backpressure-send.any.serviceworker.html?wpt_flags=h2]
   expected:
-    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if (os == "win") and (processor == "x86") and debug: OK
-    if (os == "android") and debug and not webrender: ["TIMEOUT", "ERROR"]
-    if (os == "android") and not debug and webrender: ["TIMEOUT", "ERROR"]
-    if (os == "win") and (processor == "x86_64"): OK
-    TIMEOUT
+    if (os == "win") and (processor == "x86") and not debug: ["OK", "TIMEOUT"]
+    if (os == "android") and debug and not webrender: ["OK", "ERROR", "TIMEOUT"]
+    if (os == "android") and not debug and webrender: ["OK", "ERROR", "TIMEOUT"]
   [backpressure should be applied to sent messages]
     expected: FAIL
 
 
 [backpressure-send.any.worker.html?wpt_flags=h2]
   expected:
-    if (os == "win") and (processor == "x86") and debug: OK
-    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if (os == "android") and debug and not webrender: ["TIMEOUT", "ERROR"]
-    if (os == "android") and not debug and webrender: ["TIMEOUT", "ERROR"]
-    if (os == "win") and (processor == "x86_64"): OK
-    TIMEOUT
+    if (os == "android") and debug and not webrender: ["OK", "ERROR", "TIMEOUT"]
+    if (os == "android") and not debug and webrender: ["OK", "ERROR", "TIMEOUT"]
+    if (os == "win") and (processor == "x86") and not debug: ["OK", "TIMEOUT"]
   [backpressure should be applied to sent messages]
     expected: FAIL
 
 
 [backpressure-send.any.html?wpt_flags=h2]
   expected:
-    if (os == "android") and debug and not webrender: ["TIMEOUT", "ERROR"]
-    if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if (os == "win") and (processor == "x86") and debug: OK
-    if (os == "win") and (processor == "x86_64"): OK
-    TIMEOUT
+    if (os == "win") and (processor == "x86") and not debug: ["OK", "TIMEOUT"]
+    if (os == "android") and debug and not webrender: ["OK", "ERROR", "TIMEOUT"]
   [backpressure should be applied to sent messages]
     expected: FAIL
 